pkg/registry/storage/rest/storage_storage.go

```diff
@@ -70,6 +72,14 @@ func (p RESTStorageProvider) v1beta1Storage(apiResourceConfigSource serverstorag
 	volumeAttachmentStorage := volumeattachmentstore.NewStorage(restOptionsGetter)
 	storage["volumeattachments"] = volumeAttachmentStorage.VolumeAttachment
 
+	// csinodeinfos
```

Comment:
This should be feature gated so we don't register the objects if the feature is disabled